lib/pure/collections/lists.nim

```diff
@@ -382,9 +368,16 @@ proc contains*[T](L: SomeLinkedCollection[T], value: T): bool {.inline.} =
 
   result = find(L, value) != nil
 
-proc append*[T](L: var SomeLinkedCollection[T], n: SomeLinkedNode[T] | T)
-    {.deprecated: "Deprecated since v1.6; use `add`".} =
-  add(L, n)
+proc append*[T](a: var SomeLinkedCollection[T], b: SomeLinkedNode[T] | T) =
+  ## Alias for `a.add(b)`.
```

Comment:
I aknowledge this comment `Yeah, please don't deprecate append, I don't appreciate ecosystem-damaging micro-optimizations.` but can we at least write:
```nim
## Deprecated alias for `a.add(b)`.
```

this won't cause any warning but at least will nudge users to use `add` instead of `append`, for reasons already discussed before in https://github.com/nim-lang/RFCs/issues/303#issuecomment-747787419